### DISCUSSION TITLE
Repository download size

### DISCUSSION THREAD

**Discussion by Stevepurpose** [mdn/mdn-community#445] [Content]
Goodday MDN ,While going through your docs, I  found a particular page I'd like to edit. Clicking on it takes me to a content repository on GitHub.  My question is this : when I fork the repository does it  download the entire MDN Docs or just the page I want to work on?.
 I am asking because I'd like to be aware of the internet  data bundle consumption  in megabytes or gigabytes,so that I don't get caught unawares . I have just 1.5GB currently,would that be enough to complete the fork?.

**Comment by bsmth** [comment]
Thanks for asking. The `mdn/content` repo is about 300MB (a little more on disk uncompressed) so you can expect around that much data to be transferred/used when cloning your fork:

```bash
git clone --verbose  https://github.com/mdn/content.git
# Receiving objects: 100% (654355/654355), 299.58 MiB | 6.10 MiB/s, done.
```

It is probably safe to do a [shallow clone for your fork](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone) - this comes out to around 60MB:

```bash
git clone --depth=1 --verbose  https://github.com/{your_username}/content.git
# ...
# Receiving objects: 100% (25510/25510), 57.98 MiB | 6.07 MiB/s, done.
```

Alternatively, you can use the GitHub editor to edit a page without having to download anything over your network. See [GitHub docs for this](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files#editing-files-in-another-users-repository).

I hope that helps!

**Comment by Stevepurpose** [comment]
Thanks a lot for the info. Will check out the options.
